### ISSUE TITLE
similar

### ISSUE THREAD

**Issue by gedw99** [c0olix/asyncApiCodeGen#29]
https://github.com/lerenn/asyncapi-codegen looks pretty similar this this ?

**Comment by c0olix** [comment]
True, but this ones multilingual (kind of, since it's only Java). Not saying mines better though 😉 Maybe the other one suites my purposes as well, then maybe I'll drop this one.
